docs/stable_branch.md

```diff
@@ -0,0 +1,52 @@
+# Stable Branch
+
+## Overview
+The OCS-CI team has adopted the use of a stable branch for ocs-ci. The intention
+here is to perform additional testing on code changes that are merged to our master branch
+before they are deemed stable. Once testing has passed, the new changes will be propagated
+to our stable branch and can be used for production testing or any other testing that
+requires the use of our stable branch.
+
+The initial proposal, discussion, and implementation details can be found in a
+[shared google doc](https://docs.google.com/document/d/15rqGVDEWFTFsg6FOOBeWUYYrwBa9wi0rZtdfOYi_-zg/edit).
```

Comment:
Make a comment that it's available only for RH internal accounts

Reply:
Done